tests/testnet-calls.py

```diff
@@ -0,0 +1,343 @@
+        "chainId": CHAIN_ID,
+        "gas": 550000,  # you might want to adjust this value based on the real gas consumption
+        "gasPrice": w3.to_wei('20', 'gwei'),
+        "nonce": w3.eth.get_transaction_count(sender_account.address)
+    })
+
+    try:
+    # Send escrow deposit
+        deposit_txn = send_signed_transaction(sender_account, deposit_txn).hex()
+    except ValueError as e:
+        error = str(e)
+        if "transaction underpriced" in error:
+            deposit_txn = escrow.functions.deposit(receiver, amount).build_transaction({
+                "chainId": CHAIN_ID,
+                "gas": 550000,  # you might want to adjust this value based on the real gas consumption
+                "gasPrice": w3.to_wei('20', 'gwei'),
+                "nonce": w3.eth.get_transaction_count(sender_account.address) + 1
+            })
+            deposit_txn = send_signed_transaction(sender_account, deposit_txn).hex()
+    print("txn hash deposit: ", deposit_txn)
+    check_deployed_subgraph_transaction(
+        deposit_txn, receiver, sender_account.address,
+        amount, 'deposit', SUBGRAPH_ENDPOINT
+    )
+    
+
+def thaw(sender_account, receiver, amount):
+    #escrow.functions.thaw(SIGNER, 10).transact({"from":GATEWAY})
+    thaw_txn = escrow.functions.thaw(receiver, amount).build_transaction({
+        "chainId": CHAIN_ID,
+        "gas": 550000,  # you might want to adjust this value based on the real gas consumption
+        "gasPrice": w3.to_wei('20', 'gwei'),
+        "nonce": w3.eth.get_transaction_count(sender_account.address)
+    })
+
+    # Send escrow thaw
+    print("txn hash thaw: ", send_signed_transaction(sender_account, thaw_txn).hex())
+
+def thawSigner(sender_account, signer):
+    #escrow.functions.thaw(SIGNER, 10).transact({"from":GATEWAY})
+    thaw_txn = escrow.functions.thawSigner(signer).build_transaction({
+        "chainId": CHAIN_ID,
+        "gas": 550000,  # you might want to adjust this value based on the real gas consumption
+        "gasPrice": w3.to_wei('20', 'gwei'),
+        "nonce": w3.eth.get_transaction_count(sender_account.address)
+    })    
+
+    try:
+    # Send escrow thaw
+        thaw_txn = send_signed_transaction(sender_account, thaw_txn).hex()
+    except ValueError as e:
+        error = str(e)
+        print(error)
+        if "transaction underpriced" in error:
+            thaw_txn = escrow.functions.thawSigner(signer).build_transaction({
+                "chainId": CHAIN_ID,
+                "gas": 550000,  # you might want to adjust this value based on the real gas consumption
+                "gasPrice": w3.to_wei('20', 'gwei'),
+                "nonce": w3.eth.get_transaction_count(sender_account.address) + 1
+            })
+            thaw_txn = send_signed_transaction(sender_account, thaw_txn).hex()
+    print("txn hash thaw Signer: ", thaw_txn)
+    check_subgraph_signer(SIGNER, True, True, SUBGRAPH_ENDPOINT, True)
```

Comment:
Could you name the arguments for all the `check_subgraph_` function calls? It's not super readable rn.